app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt

```diff
@@ -74,6 +75,16 @@ fun createNimbus(context: Context, url: String?): NimbusApi {
         context.settings().isFirstNimbusRun = false
     }
 
+    // These values can be used in the JEXL expressions when targeting experiments.
+    val customTargetingAttributes = JSONObject().apply {
+        // By convention, we should use snake case.
+        put("is_first_run", isFirstNimbusRun)
```

Comment:
nit: I know it's tempting to fix bugs as you go, but it's better for tracking if you file a bug and go back and fix them later (unless the bug is blocking what you are working on).

Reply:
Oh this is from #27899 -- I branched from that PR since it included the Nimbus breaking change fixes for 96.0.1